ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# Architecture
+
+The general pattern of usage is to instantiate the `ReCaptcha` class with your
+secret key, specify any additional validation rules, and then call `verify()`
+with the reCAPTCHA response and user's IP address. For example:
+
+```php
+<?php
+$recaptcha = new \ReCaptcha\ReCaptcha($secret);
```

Comment:
This appears a couple of times. Is the `\ReCaptcha\ReCaptcha` intentional (both the `\` and the double “ReCaptcha” confuse me.

Reply:
This is PHP's namespacing. A leading `\` gives the fully qualified name, i.e. namespace and class. Inside a class, you'd usually use `use Recaptcha` to reference the namespace, allowing `new Recaptcha($secret);`

In the docs and examples, where they're just single or standalone files I've been using the fully qualified namespace.

Reply:
I literally have _never_ seen that. TIL. Thanks :)